src/server/zset_family.h

```diff
@@ -34,6 +34,8 @@ class ZSetFamily {
 
   using LexInterval = std::pair<LexBound, LexBound>;
 
+  using ScoreCount = uint32_t;
+
```

Comment:
I'd suggest adding a comment explaining what `ScoreCount` means, because its not obvious from its name. Or maybe rename it to something like `TopNScored`.

Reply:
I went with your second suggestion here, I think it's more expressive